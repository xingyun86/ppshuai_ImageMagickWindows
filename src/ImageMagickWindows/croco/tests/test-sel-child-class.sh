#!/bin/sh

. global-test-vars.sh

$CSSLINT -e --user-sheet $TEST_INPUTS_DIR/sel-child-class.css --xml $TEST_INPUTS_DIR/sel-child-class.xml --xpath /test/body/div/div/p
