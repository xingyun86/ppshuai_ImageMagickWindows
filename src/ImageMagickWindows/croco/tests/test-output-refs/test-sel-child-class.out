Properties of xml element p are:
  color : aqua
=====================

